grid/booking/list-activities/profile.supr

```diff
@@ -0,0 +1,185 @@
+"""
+List Activities
+Lists activities for which a booking can be created.
+"""
+name = "booking/list-activities"
+version = "1.0.0"
+
+"""
+List Activities
+Lists activities for which a booking can be created.
+"""
+usecase ListActivities safe {
```

Comment:
Good point, but there's no demand for filtering/search *yet*, and since it can be added in a backwards-compatible way, I'd leave it for the next iteration?